PKGBUILD

```diff
@@ -1,19 +1,16 @@
 
-pkgname="libixp-hg"
+pkgname="libixp"
 pkgver=131
 pkgrel=1
-pkgdesc="The latest hg pull of libixp, a simple 9P filesystem library"
-url="http://libs.suckless.org/libixp"
+pkgdesc="a simple 9P filesystem library"
```

Comment:
Maybe start the description with a capital.

Reply:
:+1: